### PR TITLE
feat: Add support for gradle's kotlin-dsl

### PR DESCRIPTION
**Pull request by pastjean**
- [x] Ready for review
- [x] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/.github/CONTRIBUTING.md) rules
- [ ] Reviewed by Snyk internal team

#### What does this PR do?

As of gradle v5 (soon to be released), kotlin-dsl is now official. It still uses the
same old methods as gradle but the detection of those project with
snyk was broken. This patch adds the support for gradle kotlin-dsl

#### How should this be manually tested?
Create a `build.gradle.kts` file in a directory with the following content:

```kotlin
plugins {
    kotlin("jvm")
}

dependencies {
    implementation(kotlin("stdlib"))
}
```

#### Any background context you want to provide?

Kotlin-dsl: https://github.com/gradle/kotlin-dsl/

News here: https://github.com/gradle/gradle/releases/tag/v5.0.0-RC2#kotlin-dsl-1.0

### CODE REVIEW

**Comment by pastjean** [comment]
@michael-go , 1st sorry for the random ping,

How can I ensure this gets someone from snyk's approval ?

**Comment by pastjean** [comment]
Thanks for the quick answer. 🙌

File: test/acceptance/cli.acceptance.test.js [hunk 1]
```diff
@@ -1970,6 +1996,7 @@ test('`protect` for unsupported package managers', function (t) {
     { file: 'pip-app/requirements.txt', type: 'Python' },
     { file: 'sbt-app/build.sbt', type: 'SBT' },
     { file: 'gradle-app/build.gradle', type: 'Gradle' },
+    { file: 'gradle-kotlin-dsl-app/build.gradle', type: 'Gradle' },
```
Comment (tbroyer) [inline test/acceptance/cli.acceptance.test.js]: Should this be `gradle-kotlin-dsl-app/build.gradle.kts`?

Comment (pastjean) [reply test/acceptance/cli.acceptance.test.js]: Yeah, what a mistake.

**Comment by pastjean** [comment]
@tbroyer is everything good now ? Or do I need more

**Comment by lili2311** [comment]
Hi! Just to keep you updated this is on my teams radar, there is some upstream work needed to make this miserable which we are working on :)

I see some tests are failing, kicked off the build again seems like a fluke

**Comment by Kirill89** [comment]
Tests are broken because travis not pass private environment variables to forked code.
I created [different PR](https://github.com/snyk/snyk/pull/395) and merged your code into it.
@pastjean thank you for your help!

**Pull request merged**